src/atoms/TextArea/TextArea.native.js

```diff
@@ -79,6 +82,7 @@ TextArea.propTypes = {
 
 TextArea.defaultProps = {
   placeholder: '',
+  autoFocus: false,
```

Comment:
Can we keep it as 
```suggestion
  autoFocus: undefined,
```

This way the existing snapshots won't change either. It's internally false if undefined anyway